core/tests/ledger_cleanup.rs

```diff
@@ -331,23 +309,23 @@ mod tests {
         false
     }
 
-    /// The ledger cleanup compaction test which can also be used as a benchmark
+    /// The ledger cleanup  test which can also be used as a benchmark
     /// measuring shred insertion performance of the blockstore.
     ///
     /// The benchmark is controlled by several environmental arguments.
     /// Check [`get_benchmark_config`] for the full list of arguments.
     ///
     /// Example command:
     /// BENCHMARK_SLOTS=1000000 BATCH_SIZE=1 SHREDS_PER_SLOT=25 NUM_WRITERS=8 \
-    /// PRE_GENERATE_DATA=false cargo test --release tests::test_ledger_cleanup_compaction \
+    /// PRE_GENERATE_DATA=false cargo test --release tests::test_ledger_cleanup \
     /// -- --exact --nocapture
     #[test]
-    fn test_ledger_cleanup_compaction() {
+    fn test_ledger_cleanup() {
```

Comment:
I went back and forth on whether to rip this test out (and thus this whole file):
- The bit about testing cleanup is now not really exercised here. Our cleaning methods (delete_files_in_range() for level OR fifo) both require a sizable amount of data to be inserted in order to see cleanup actually reclaim anything (on account of them operating on entire SST's). So, we'd have to make sure we pushed enough data to get several memtable flushes.
- We had used this as a bench previously for max shred insertion rate; however, I think the usefulness of that aspect may have gone away when merkle shreds were introduced and we could no longer hack the slot in quickly (thus making shred generation the bottleneck).

Reply:
Yep, agree that we can no longer abuse this test to do benchmarks due to the merkle shreds change.

We can decide the destiny of this file in a different PR.

Reply:
> We can decide the destiny of this file in a different PR.

Sounds good to me